packages/react-native-oursky/src/VerifyOTP.js

```diff
@@ -140,10 +138,19 @@ class VerifyOTP extends React.PureComponent<Props & TextInputRefProps, State> {
     }
   };
 
+  focus = () => {
+    if (this.textInputRef && this.textInputRef.current) {
+      this.textInputRef.current.focus();
+    }
+  };
+
   renderCodeBox = () => {
     const { codeBoxStyle, codeBoxTextStyle } = this.props;
     return (
-      <View style={defaultStyles.codeBoxContainer}>
+      <TouchableOpacity
```

Comment:
This make bad UX as 4 `CodeBox`es' opacity changed together when pressing either one of them.

1. Change to use `TouchableWithoutFeedback`
2. Wrap `CodeBox` with `TouchableWithoutFeedback` one by one. i.e. 4 different `TouchableWithoutFeedback`s